src/foam/dao/MDAO.java

```diff
@@ -110,6 +110,13 @@ public FObject find_(X x, Object o) {
 
     if ( o == null ) return null;
 
+    // Temporary until DAO supports find_(Predicate) directly
```

Comment:
Is this needed? Who is calling this method directly?

Reply:
I closed this PR